src/SourceWatcherFuncApp/NCloudSourceWatcher.cs

```diff
@@ -142,17 +156,43 @@ public NCloudSourceWatcher(IConfigurationRoot configurationRoot, IGithubService
                 updatetasks.Add(storageService.LoadDataToTable(deletedRow, ""));
             });
 
+            totalDeletes = deletedIds.Count;
+
             currentIds.ForEach(currentDetector =>
             {
                 currentDetector.IsDisabled = false;
                 log.LogInformation($"Marking {currentDetector.RowKey} as active");
                 updatetasks.Add(storageService.LoadDataToTable(currentDetector, ""));
             });
 
-            await Task.WhenAll(updatetasks);
-            log.LogInformation($"C# Timer trigger function finished at: {DateTime.Now}");
+            await Task.WhenAll(updatetasks).ConfigureAwait(false);
 
-            return new OkObjectResult("");
+            var response = new CallbackResponse
+            {
+                TotalCreates = totalCreates,
+                TotalDeletes = totalDeletes,
+                TotalUpdates = totalUpdates,
+                TotalSuccess = totalSuccess,
+                TotalFailures = totalFailures
+            };
+
+            if (!string.IsNullOrWhiteSpace(config["CallbackUrl"]))
+            {
+                log.LogInformation($"Send results to {config["CallbackUrl"]}");
+                var content = new StringContent(JsonConvert.SerializeObject(response), Encoding.UTF8, @"application/json");
+                await httpClient.PostAsync(config["CallbackUrl"], content).ConfigureAwait(false);
```

Comment:
What is the callbackUrl here?

Reply:
So this is for sending stats to any HTTP service that want's to know about these periodic syncs. I have a LogicApp on the other end that is getting called by this FunctionApp via this callback URL.